tests/ci/docker_images/linux-x86/build_legacy_images.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash -ex
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+if [ -n "$1" ]; then
+  docker_name="$1"
+else
+  docker_name='ubuntu-7.10_gcc-4.1x'
+fi
+
+# `ubuntu-7.10-gutsy_gcc-4.1x` was rebuilt from an iso image
+# from: https://old-releases.ubuntu.com/releases/gutsy/
+# The docker tar file is checked into our team account.
```

Comment:
In which s3 bucket?

Reply:
I've gone with a different method now. The new steps are what I originally did to create the image. Now the script downloads the iso image and builds a docker image from that. This should be more sustainable than fixating the `build-linux-docker-img` Codebuild project to a specific s3 bucket.
I've tested the `build-linux-img` cdk command on my account and the new steps works.